Tests/LibJS/test-js.cpp

```diff
@@ -58,6 +58,40 @@ TESTJS_GLOBAL_FUNCTION(get_weak_map_size, getWeakMapSize)
     return JS::Value(weak_map->values().size());
 }
 
+TESTJS_GLOBAL_FUNCTION(mark_as_garbage, markAsGarbage)
+{
+    if (!vm.argument(0).is_string()) {
+        vm.throw_exception<JS::TypeError>(global_object, JS::ErrorType::NotA, "String");
```

Comment:
```suggestion
        vm.throw_exception<JS::TypeError>(global_object, JS::ErrorType::NotAString, vm.argument(0).to_string_without_side_effects();
```

note that `NotA` is terribly named and outputs "Not a String object", which is incorrect.